s/elements/stick/element.ts

```diff
@@ -21,24 +22,23 @@ export class NubStick extends MagicElement {
 	realize() {
 		const {use} = this
 
-		const [, setTrackingMouse, getTrackingMouse] =
-			use.state(false)
-
-		const [, setTrackingTouchId, getTrackingTouchId] =
+		const [, setTrackingPointerId, getTrackingPointerId] =
 			use.state<number | undefined>(undefined)
 
-		const [styleTransforms, setStyleTransforms] =
-			use.state({stick: "", understick: ""})
+		const [vector, setVector] = use.state({
+			x: 0,
+			y: 0
+		})
 
 		const starters: StickStarters = {
-			setStyleTransforms,
-			getTrackingMouse,
-			setTrackingMouse,
-			getTrackingTouchId,
-			setTrackingTouchId,
+			setVector,
+			setTrackingPointerId,
+			getTrackingPointerId,
 			query: () => ({
-				base: this.shadowRoot!.querySelector(".base")!,
-				stick: this.shadowRoot!.querySelector(".stick")!,
+				base: this.shadowRoot?.querySelector("nub-stick-graphic")!
+					.shadowRoot?.querySelector("[part='base']")!,
+				stick: this.shadowRoot?.querySelector("nub-stick-graphic")!
+					.shadowRoot?.querySelector("[part='stick']")!
```

Comment:
drilling through shadow boundaries is a bad pattern, because it breaks encapsulation.

instead, the stickpad needs to interact with nub-stick-graphic's public interface.

we may need to redesign the nub-stick-graphic's public interface to fix this.